content/intro-to-storybook/svelte/en/creating-addons.md

```diff
@@ -0,0 +1,473 @@
+---
```

Comment:
I'm guessing this file is unchanged as it is not in Svelte (of course). LMK if not.

Reply:
The creating addons was moved from the existing one for the react version and was not changed.  If you don't mind i'm going to mark this as resolved.